static/sass/styles.scss

```diff
@@ -24,7 +25,11 @@ $p-small-lh-diff: map-get($line-heights, default-text) -
 @import "pattern_strip";
 @import "pattern_tabs";
 @import "pattern_search-panel";
+@import "pattern_search-form";
 
+@include vanilla;
+@include vf-p-strip;
```

Comment:
If you include `vanilla` you don't need to include anything else from it. It will have everything that comes in the framework.